yt/sample_data_registry.json

```diff
@@ -726,7 +726,7 @@
   "maestro_xrb_lores_23437.tar.gz": {
     "hash": "a17b63609a2c50a829e26420f093ee898237b1ef8256ef38ee8be56a6f824460",
     "load_kwargs": {},
-    "load_name": "",
+    "load_name": null,
```

Comment:
In case you're wondering what I did here, this doesn't affect the program functionally, but it was the only dataset left with an empty string for a load_name, so I figure we should embrace `null` as a signal that the the directory from the tarball _is_ the loadable path (as opposed to the general case where we need to append a load_name to form a complete loadable path).
There might be other non-functional datasets left in the repo using `"load_name": null` and which fail in different ways. I'll fix those in a followup patch if needed.